nix/chainmain.nix

```
@@ -0,0 +1,8 @@
+{pkgs ? import ./default.nix {} }:
+let
+  released = (import (builtins.fetchTarball "https://github.com/crypto-org-chain/chain-main/archive/v3.1.1.tar.gz") { }).chain-maind;
+in
+ pkgs.symlinkJoin {
+    name = "chainmain";
+    paths = [ released ];
+ }
```

```suggestion
(import (builtins.fetchTarball "https://github.com/crypto-org-chain/chain-main/archive/v3.1.1.tar.gz") { }).chain-maind
```

It seems don't need the extra stuff?

---

ok, i'll check it

---

done